tests/testthat/test-rr_journal_name.R

```diff
@@ -2,17 +2,17 @@ context("rr_journal_name")
 
 test_that("rr_journal_name() works", {
   use_cassette("rr_journal_name", {
-    res <- rr_journal_name("Journal of Geology")
+    res <- rr_journal_name("Journal of Biogeography")
 
     expect_is(res, "data.frame")
 
     expect_named(res, c("title", "provided_issn", "issn", "romeocolour",
                         "preprint", "postprint",
                         "pdf", "pre_embargo", "post_embargo", "pdf_embargo"))
-    expect_equal(res$issn, "0022-1376")
+    expect_equal(res$issn, "0305-0270")
     expect_is(res$romeocolour, "character")
     expect_is(res$preprint, "character")
-  })
+  }, preserve_exact_body_bytes = TRUE)
```

Comment:
I don't understand why use changing the value of `preserve_exact_body_bytes` here. If it is truly needed, it would be good to add a comment explaining why.

Reply:
True, this should be removed because it was from tests I was doing